docker/runtime/nodejs/kubeless.js

```diff
@@ -99,8 +99,13 @@ function modExecute(handler, req, res, end) {
 }
 
 function modFinalize(result, res, end) {
+    if(res.finished){
+        return end();
+    }
+
     switch(typeof result) {
         case 'string':
+        case 'undefined':
```

Comment:
would `res.end(undefined)` work? Does it happen the same with `null`?

Reply:
`res.end()` and `res.end(undefined)` give exactly the same response.

`null` is a special case, and will be handled by the 'object' case in this switch statement. It'll result in a JSON response containing literally the 4 bytes of 'null', which is valid JSON (see the comment on my commit for a link to the RFC)

Reply:
Maybe like this:
```
function modFinalize(result, res, end) {
    if (!res.finished) switch(typeof result) {
        case 'string':
            res.end(result);
            break;
        case 'object':
            if (result === null) {
                res.end();
            } else {
                res.json(result); // includes res.end()
            }
            break;
        case 'undefined':
            res.end();
            break;
        default:
            res.end(JSON.stringify(result));
    }

    end();
}
`´`

Reply:
@cscheiber we could do that but it seems a bit odd - given that `null` is valid JSON, should we not return it as JSON instead of an empty response? It's not a big deal to me whichever we decide on, but it seems like a function would return null on purpose, so we should send it down to the browser?

Reply:
BTW, `res.end()` and `res.end(undefined)` do not behave completely the same; inside the function `arguments.length` will be either 0 or 1

Reply:
@cscheiber sorry deleted my last comment as it referred to your unedited comment!

I'm happy to make this change if the style is more consistent with your codebase.